content/docs/cli-reference/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: cli-reference
+title: CLI-Reference
```

Comment:
why is there a "-" included still? arent these 2 words?

Reply:
Indeed :-) Corrected.